### DISCUSSION TITLE
How to update repo Languages info

### DISCUSSION THREAD

**Discussion by lslzl3000** [community/community#54929] [Discussions] [answered]
### Select Topic Area

Question

### Body

Our public repo has a wrong Languages Statistics info, and it's been a long time since last update.
![Screenshot 2023-05-09 at 16 39 05](https://user-images.githubusercontent.com/3398981/237042253-e1f36e04-26ec-4961-8c53-02e9ab471eb3.png)

How to trigger a update? We are committing and PR everyday, including changes of `README.md`, but nothing changes

**Comment by murilobarbosaa** [comment]
You can delete the code, commit with the empty readme, and then put the code again to commit with it. it will update the stats

**Comment by airtower-luna** [comment, marked as answer]
Lingust (the language detection system GitHub uses) excludes documentation (including `README.md`) and a few other things by default. See their [troubleshooting guide](https://github.com/github-linguist/linguist/blob/master/docs/troubleshooting.md) for more about that and how to override the defaults if you want to include them anyway.
